Styles/docs.js

```diff
@@ -0,0 +1,20 @@
+function copyFunction() {
+    const copyText = document.getElementById("myInput").textContent;
+    const textArea = document.createElement("textarea");
+    textArea.textContent = copyText;
+    document.body.append(textArea);
+    textArea.select();
+    document.execCommand("copy");
+    textArea.remove();
+    copiedMsg();
+  }
+  
+  document.getElementById("button").addEventListener("click", copyFunction);
+  
+  function copiedMsg() {
```

Comment:
For some reason, the copy option isn't working when clicked.

Reply:
I tried debugging it the whole evening. I have a workaround but I just found a better solution. I'll update it tomorrow first thing.